### DISCUSSION TITLE
Output displacement at a node with GeneratedMesh

### DISCUSSION THREAD

**Discussion by selarem** [idaholab/moose#22844] [Q&A Modules: Solid mechanics] [answered]
Hi,
I would like to make a force-displacement curve. 
I can have the force reactions at a boundary but I would like to output (csv table the displacement at a node from the same boundary).
When the mesh comes from a file, I could see the number of the concerned node and make postprocessing with NodalVariableValue to output the displacements. However, with GeneratedMesh, I have not the node number.
Any suggestion ?
Thanks in advance

sab

**Comment by maxnezdyur** [comment]
Would something like `NodalExtremeValue` work for you? It can give the max or min of a value from the nodes on a boundary.

**Reply by GiudGiud** [reply]
You could also output all the nodal values and postprocess from there. 
The NodalValueSampler is a VPP that does that

**Comment by selarem** [comment, marked as answer]
This is the most appropriate way to do it.
Thanks
